front-end/src/routes/dashboard/containers/dashboard.js

```diff
@@ -1,10 +1,11 @@
 import React,{Component} from 'react';
-import Header from '../../../components/common/header/Actions/headerActions'
```

Comment:
@evelinm What is the reasoning on the removal of this line? Just curious

Reply:
I deleted it because we didn't need it, I just moved the </header> tag up on a file ( forgot which one) so we didn't need to add this line to every route. Not sure if that makes sense.